www/citing.rst

```diff
@@ -24,14 +23,14 @@ Andrew R. J. Nelson, Eric Jones, Robert Kern, Eric Larson, CJ Carey,
 İlhan Polat, Yu Feng, Eric W. Moore, Jake VanderPlas, Denis Laxalde,
 Josef Perktold, Robert Cimrman, Ian Henriksen, E.A. Quintero, Charles R Harris,
 Anne M. Archibald, Antônio H. Ribeiro, Fabian Pedregosa, Paul van Mulbregt,
-and SciPy 1.0 Contributors. (2019) **SciPy 1.0--Fundamental Algorithms
-for Scientific Computing in Python**. *preprint* arXiv:1907.10121_
+and SciPy 1.0 Contributors. (2020) **SciPy 1.0: Fundamental Algorithms
+for Scientific Computing in Python**. *Nature Methods*, in press.
 
 Here's an example of a BibTeX entry:
 
 ::
 
-    @ARTICLE{2019arXiv190710121V,
+    @ARTICLE{2020-NMETH,
```

Comment:
this is a little non-specific, could you make it something like `2020SciPy-NMeth`?

Reply:
Ok, I've used that suggestion now.